index.js

```diff
@@ -10,11 +10,8 @@ module.exports = function(out, options) {
 
   var onFile = function(file) {
     files.push(file);
-    var path = file.path;
-    if (!options.absolute) {
-      path = path.replace(new RegExp('^' + process.cwd() + '/'), '');
-    }
-    filePaths.push(path);
+    var path = options.absolute ? file.path : file.relative;
```

Comment:
I think is confusing because `test.file` is in the same folder with test script.